tests/src/Core/Config/PreloadPConfigurationTest.php

```diff
@@ -108,16 +108,6 @@ public function testGetWithRefresh(int $uid, $data)
 		parent::testGetWithRefresh($uid, $data);
 	}
 
-
-	public function testGetWrongWithoutDB()
```

Comment:
Can you check why the `PConfigurationTest->testGetWrongWithoutDB()` doesn't need the `isConnected` method anymore oO

Reply:
You've been mixing two different invalid UID parameter type, either `0` or `false`. In the first case, it would have gone past your `is_int()` test, but not anymore, which returns immediately.

Reply:
ah .. I see